pkg/systemd/hwinfo.jsx

```diff
@@ -24,61 +24,195 @@ import React from "react";
+        return (
+            <table className="info-table-ct wide-split-table-ct">
+                { info.type &&
+                <tbody>
                     <tr>
-                        <th>{ _("BIOS") }</th>
-                        <td>{ info.bios_vendor }</td>
+                        <th>{ _("Type") }</th>
+                        <td>{ info.type }</td>
                     </tr>
                     <tr>
-                        <th>{ _("BIOS version") }</th>
-                        <td>{ info.bios_version }</td>
+                        <th>{ _("Name") }</th>
+                        <td>{ info.name }</td>
                     </tr>
                     <tr>
-                        <th>{ _("BIOS date") }</th>
-                        <td>{ moment(info.bios_date).isValid() ? moment(info.bios_date).format('L') : info.bios_date }</td>
+                        <th>{ _("Version") }</th>
+                        <td>{ info.version }</td>
                     </tr>
-                </React.Fragment>
-            }
-            { info.nproc !== undefined &&
-                <tr>
-                    <th>{ _("CPU") }</th>
-                    <td>{ (info.nproc > 1) ? `${info.nproc}x ${info.cpu_model}` : info.cpu_model }</td>
-                </tr>
+                </tbody>
+                }
+                <tbody>
+                    { info.bios_vendor &&
+                    <React.Fragment>
+                        <tr>
+                            <th>{ _("BIOS") }</th>
+                            <td>{ info.bios_vendor }</td>
+                        </tr>
+                        <tr>
+                            <th>{ _("BIOS version") }</th>
+                            <td>{ info.bios_version }</td>
+                        </tr>
+                        <tr>
+                            <th>{ _("BIOS date") }</th>
+                            <td>{ moment(info.bios_date).isValid() ? moment(info.bios_date).format('L') : info.bios_date }</td>
+                        </tr>
+                    </React.Fragment>
+                    }
+                    { info.nproc !== undefined &&
+                    <React.Fragment>
+                        <tr>
+                            <th>{ _("CPU") }</th>
+                            <td>{ (info.nproc > 1) ? `${info.nproc}x ${info.cpu_model}` : info.cpu_model }</td>
+                        </tr>
+                        { onSecurityClick !== undefined &&
+                        <tr>
+                            <th>{ _("CPU Security") }</th>
+                            <td>{ (this.state.allowed && <a role="link" onClick={onSecurityClick}>{ _("Mitigations") }</a>) ||
+                                <OverlayTrigger overlay={ <Tooltip id="tip-cpu-security">{ cockpit.format(_("The user $0 is not permitted to change cpu security mitigations"), permission.user ? permission.user.name : '') }</Tooltip> }><span>{ _("Mitigations") }</span></OverlayTrigger> }
```

Comment:
Please move the contents of this `<td>` into a variable at the top, this is a bit too unwieldy. Also please break the long line into multiple ones (you can move <span> or the contents to a new one)